### ISSUE TITLE
Examples Not Being HTML Escaped Correctly (Solution Included)

### ISSUE THREAD

**Issue by OptrixAU** [apidoc/apidoc#606]
In any example section (for instance, @apiSuccessExample or @apiErrorExample), any XML response will have issues because the < and > characters are not translated into the HTML equivalents '& l t ;' and '& g t ;'.

This could be resolved in a custom template, but it really **should** work out-of-the-box with the existing template.

Below is a very dodgy fix for the issue...

In **apidoc-core/lib/parsers/api_example.js**

Replace the 'return' statement with the following...
   ```
 Encoder = require('node-html-encoder').Encoder;
    var encoder = new Encoder('entity');
		
    return {
        title  : title,
        content: encoder.htmlEncode(unindent(text)),
        type   : type || 'json'
    };
```

This quite simply performs a basic string replace on the two symbols. 

This requires a new dependency - **node-html-encoder**, which would correctly deal with a range of other characters, such as double-quotes.

**Comment by noahadams** [comment]
@OptrixAU I believe this is a dupe of #588 and fixed in #589. Want to see if the code in #589 fixes your issue?

HTML entity escaping is already handled by Handlebars, but examples are being explicitly templated without escaping.

**Comment by rottmann** [comment]
Thanks, merged pull request.
